src/components/Home.js

```diff
@@ -0,0 +1,137 @@
+        <TopCard
+          backgroundImage={Map1}
+          location="USA"
+          views="890 Views"
+          footerText="Stats by air preassure"
+          toggleSearch={toggleSearch}
+          isSearchVisible={isSearchVisible}
+          setSearchKeyword={setSearchKeyword}
+        />
+        <Container fluid data-testid="content-card">
+          <Row xs={1} sm={2} md={2} lg={2} className="g-4 no-gutters p-2">
+            {filteredLocations.length > 0 ? (
+              filteredLocations.map((location, index) => {
+                const measurementKey = uuidv4();
+                const truncatedTitle = location.location.length > 20 ? `${location.location.slice(0, 20)}...` : location.location;
+                return (
+                  <Col key={uuidv4()} xs={6} md={6}>
+                    <Link to={`/details/${location.location}/${uuidv4()}`} style={{ textDecoration: 'none' }}>
+                      <Card className={`content-card flex-container flex-column bold no-border flex-end ${index % 2 === 1 ? 'darker' : ''}`} style={{ backgroundImage: `url(${Map2})` }}>
+                        <Card.Header className="position-absolute top-0 end-0">
+                          <FontAwesomeIcon icon={faCircleRight} style={{ color: '#fff' }} />
+                        </Card.Header>
+                        <Card.Body
+                          style={{ display: 'flex', flexDirection: 'column', fluid: true }}
+                          className="card-content flex-container flex-column"
+                        >
+
+                          <div className="card-info-container flex-container flex-column">
+                            <Card.Title style={{ width: '6rem' }} className="title-text white-text bold ellipsis- multiline-2 text-wrap">
+                              {' '}
+                              {truncatedTitle.toUpperCase()}
+                            </Card.Title>
+                          </div>
+                          <div className="measurements-text white-text wrap-break">
+                            {location.measurements.map((metric) => {
+                              if (airQualityParameters.includes(metric.parameter)) {
+                                return (
+                                  <div key={measurementKey} className="measurements-item">
+                                    {metric.parameter}
+                                    {' '}
+                                    :
+                                    {' '}
+                                    {metric.value}
+                                    {' '}
+                                  </div>
+                                );
+                              }
+                              return null;
+                            })}
+                          </div>
+
+                        </Card.Body>
+                      </Card>
+                    </Link>
+                  </Col>
+                );
+              })
```

Comment:
- [ ] The design for the items on the Homepage should follow the alternating color patterns as we have on the design. All the darker colors are on one side, but the design requires that you alternate the darker and lighter colors. Kindly do that.

  Your design without alternating color pattern | Required Design with alternating color pattern
  :---:|:---:
  ![image](https://github.com/RileyManda/air-quality/assets/36542986/c4cc4731-0f44-4efe-9753-1ea3fd3bb63d) | ![uiww](https://github.com/RileyManda/air-quality/assets/36542986/9e8c17f4-7aea-4732-a3c6-77c4daaa4354)